tests/dummy/app/templates/docs/between.md

```diff
@@ -0,0 +1,107 @@
+# Animating Between Components
+Sprites can travel between two separate lists that know about each other. When a sprite starts in one list and ends in another, the two lists know that the sprite came from one and went to the other. This also applies in the opposite direction. So, sprites always have an initial and a final destination. Two lists identify when a sprite is animating between them by checking the states of that sprite. 
```

Comment:
This needs a "why" explanation. Why would someone want to use this or know about it?